### DISCUSSION TITLE
Issues when launching SEB via downloaded config file

### DISCUSSION THREAD

**Discussion by ChuckJohnson-SC** [SafeExamBrowser/seb-win-refactoring#981] [Q&A]
Greetings,

I have a student using a Windows SurfaceBook running Windows 11 (not in S mode that I can tell) that when opening SEB via a config.seb file, a dialog appears stating that the configuration needs to be changed and gives the option to Uninstall SEB or cancel.

I downloaded the Verificator for Windows and opened it hoping that it would identify an issue. When I open the verificator app, it recognizes the most recently downloaded config.seb file, and I request to verify. The application does all of its debug checks successfully and then proceeds to open SEB successfully and the student can take the quiz. I then checked to see if SEB will open the .seb file properly, but it once again alerts me that the configuration needs to be modified and I have the option to uninstall or cancel.

I have completely uninstalled SEB and reinstalled it, but the results are the same. I have worked with 100s, if not 1,000s of students and this is the first time I have encountered this issue. Has anyone else had this issue, and if so, how did you resolve it?

Any and all thoughts/ideas on how to resolve this issue are appreciated.

Thanks,

Chuck

**Comment by dbuechel** [comment]
Could you please post an image of the dialog you're seeing? If it is the setup which is opened, then there might be an issue with the file association for SEB configuration files.
